internal/run/wrapper.go

```diff
@@ -47,6 +50,41 @@ func (a *API) ReplaceService(namespace, serviceID string, svc *run.Service) (*ru
 	return a.Client.Namespaces.Services.ReplaceService(serviceName, svc).Do()
 }
 
+// ServicesWithLabelSelector gets services filtered by a label selector.
+func (a *API) ServicesWithLabelSelector(namespace string, labelSelector string) ([]*run.Service, error) {
+	parent := fmt.Sprintf("namespaces/%s", namespace)
+
+	servicesList, err := a.Client.Namespaces.Services.List(parent).LabelSelector(labelSelector).Do()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to filter services by label selector")
+	}
+
+	return servicesList.Items, nil
+}
+
+// Regions gets the supported regions for the project.
+func Regions(project string) ([]string, error) {
+	if len(regions) != 0 {
+		return regions, nil
+	}
+
+	client, err := run.NewService(context.Background())
```

Comment:
avoid `context.Background` pass `ctx` here. That's why we have ctx, if we don't use it, it's pointless.